### ISSUE TITLE
No module named lexers (After update to spyder 3.0)

### ISSUE THREAD

**Issue by aanastasiou** [spyder-ide/spyder#3470]
## Problem description
Having updated to spyder 3.0 via `pip`, I attempted to start it up. The initial error I received was that there was no module named paths. I fixed that by upgrading my iPython. Subsequently, attempting to start up Spyder now gives a "No module named lexers". I suspected this was because of pygments, but pygments is at its latest version. 

**What steps will reproduce the problem?**
I am not sure what steps would reproduce the problem because it probably depends very much on the system composition:

* [Here is the error](http://pastebin.com/BKDvTqxP)
* [Here is the output of my `pip freeze`](http://pastebin.com/ZXgYNLCG)

Under these conditions, issuing a `spyder` at a command line does nothing.

Running python 2.7.6 at the moment.

**What is the expected output? What do you see instead?**
Spyder should start, standard error trace instead.

## Versions and main components

* Spyder Version: 3.0.0
* Python Version: 2.7.6
* Operating system: Linux (Ubuntu 15.04)

## Dependencies
N/A

**Comment by ccordoba12** [comment]
It seems the problem is you don't `setuptools` installed.

**Comment by aanastasiou** [comment]
Thank you for letting me know, `pip list|egrep 'setuptools'` says `setuptools (28.0.0)`. Just realised I uploaded my `pip freeze` output rather than `pip list`.

**Comment by ccordoba12** [comment]
Then I don't know how else to help you, sorry.

**Comment by aanastasiou** [comment]
Possibly sorted this out by doing the following:

I noticed that despite the fact that I had upgraded ipython to 5 via pip, 1.2.1 was still getting activated. So I removed the ipython package that I had installed long time ago via apt-get and of course the system now started activating ipython 5. Subsequently, resetting the Spyder environment and starting up again started the application without problems.
